### PR TITLE
remove manifest service

### PR DESCRIPTION
**Pull request by macat**

### CODE REVIEW

File: pkg/deployment/manifests.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import (
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+)
+
+// LoadManifestFolder returns a map of name:Manifest for each file in root
+func LoadManifestFolder(root, ext string) (map[string]*Manifest, error) {
+	var mm = make(map[string]*Manifest)
+	extRX, err := regexp.Compile(`\.[^.]+$`)
```
Comment (poopoothegorilla) [inline pkg/deployment/manifests.go]: do we not care about this error?

Comment (macat) [inline pkg/deployment/manifests.go]: yes we do!

**Comment by poopoothegorilla** [comment]
+1 looks good 💃

**Comment by bobbytables** [comment]
:shipit:

**Pull request merged**